src/routes/User.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  import { Link } from 'svelte-routing';
+
+</script>
+
+<style>
+  button {
+    height: 300px;
+    width: 500px;
+    border-bottom: 20px;
+    color: #921C2B;
+  }
+</style>
+
+<section>
```

Comment:
Nice foresight on this. I take it this will route to different shelves in later iterations?